code/datums/diseases/advance/symptoms/sensory.dm

```diff
@@ -15,94 +15,51 @@ Bonus
 
 //////////////////////////////////////
 */
-/datum/symptom/sensory_restoration
-	name = "Sensory Restoration"
+/datum/symptom/mind_restoration
+	name = "Mind Restoration"
 	stealth = -1
 	resistance = -4
 	stage_speed = -4
 	transmittable = -3
 	level = 5
 	severity = 0
 
-/datum/symptom/sensory_restoration/Activate(var/datum/disease/advance/A)
+/datum/symptom/mind_restoration/Activate(datum/disease/advance/A)
 	..()
 	if(prob(SYMPTOM_ACTIVATION_PROB * 3))
 		var/mob/living/M = A.affected_mob
 		var/datum/reagents/RD = M.reagents
 
-		switch(A.stage)
-			if(2)
-				if(RD.check_and_add("oculine", 10, 10))
-					to_chat(M, "<span class='notice'>Your hearing feels clearer and crisp.</span>")
-			if(3)
-				if(RD.check_and_add("antihol", 10, 10))
-					to_chat(M, "<span class='notice'>You feel sober.</span>")
```

Comment:
Personally not a fan of removing the `to_chat` messages, why remove them when they're harmless and easily serve the same function on the refactored mind restoration?

Reply:
So the reason why  is how this proc works; it previously just added a reagent, and if hte proc returned `TRUE`, then it would give the message, so it's not going to happen very often.

Adjust mob procs directly is always going to return `TRUE` (or not even return a value at all in most cases), so doing a `if(X()) message()` isn't really a viable option, so it'd just spam.

Reply:
Aye fair enough, that makes sense